drivers/video/video_stm32_dcmi.c

```diff
@@ -302,11 +315,32 @@ static int video_stm32_dcmi_dequeue(const struct device *dev, struct video_buffe
 {
 	struct video_stm32_dcmi_data *data = dev->data;
 
+	if (data->snapshot_mode) {
+		/* See if we were already called and have an active buffer */
+		if (!data->vbuf) {
```

Comment:
```suggestion
		if (data->vbuf == NULL) {
```
Ditto 2 lines below.

Reply:
done - Note I had it like that before, but the code checker in strict mode wanted the change to what
I am changing back.